Changes for NewsML-G2 version 2.35

bquinn opened the pull request:
* Updating dev schema to 2.35. Fixes iptc/news-architecture-dev#157

bquinn committed:
Updating dev schema to 2.35. Fixes iptc/news-architecture-dev#157

bquinn committed:
Change for 2.35 - implements G2-CR00220 - fixes iptc/news-architecture-dev#156

davecompton commented on specification/NewsML-G2_2.35-spec-All-Power.xsd:
@bquinn 
NewsML-G2 2.35: dataMining added ... that was in 2.34
Same change/comment in other files ...

davecompton commented on examples/NewsML-G2_2.35-spec-All-Power.xsd:
@bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

davecompton commented on specification/XML-Schema_FileVersion_1/NewsML-G2_2.35-spec-All-Power_1.xsd:
@bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

davecompton commented on specification/XML-Schema_FileVersion_1/NewsML-G2_2.35-spec-Framework-Power_1.xsd:
@bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

davecompton commented on specification/individual/NewsML-G2_2.35-spec-Framework-Power.xsd:
@bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

davecompton commented on tests/schema_versions/NewsML-G2_2.35-spec-All-Power.xsd:
@bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

bquinn committed:
Fixed version number in comment: "NewsML-G2 2.35: dataMining added" should be 2.34

davecompton reviewed (approved):
Looks good

bquinn commented on dev-schema/NewsML-G2dev_0.10_nar235.xsd:
```diff
@@ -0,0 +1,56 @@
+		<xs:documentation>Created by Michael Steidl</xs:documentation>
+		<xs:documentation>Revised by Brendan Quinn</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://iptc.org/std/nar/2006-10-01/" schemaLocation="NewsML-G2_2.34-spec-All-Power.xsd"/>
```
should be 2.35

bquinn commented on tests/schema_versions/NewsML-G2dev_0.10_nar235.xsd:
```diff
@@ -0,0 +1,56 @@
+		<xs:documentation>Created by Michael Steidl</xs:documentation>
+		<xs:documentation>Revised by Brendan Quinn</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://iptc.org/std/nar/2006-10-01/" schemaLocation="NewsML-G2_2.34-spec-All-Power.xsd"/>
```
should be 2.35

bquinn committed:
Update referred version in dev schema

bquinn committed:
A couple more dataMining references that needed to be updated

bquinn committed:
Fixing complexContent -> simpleContent in more places:

on NewsMessage:
- sender
- origin
(Note timestamp had already been changed, with the comment:
<!-- NAR 1.2 erratum 1 correction: on timestamp, complexContent replaced by simpleContent -->)
- destination
- channel

in DescriptiveMetadataGroup:
- keyword
- creditline

in concept:
- ConceptNameType

in eventDetails:
- start
- end

in contactInfo:
- AddressType
- AddressType/line

in pubHistory/published:
- timestamp

After these changes, XMLSpy no longer gives warnings upon opening the schema.

bquinn commented:
I tried loading the schema into XMLSpy on Friday, and discovered that there were many other places that also needed to have complexContent changed to simpleContent to avoid XML Spy warnings and schema re-writes. So I have now applied those changes, and XMLSpy now opens the schema with no warnings. (It still re-writes the schema but all changes are to whitespace and formatting.)

I have updated the Change Request accordingly: http://dev.iptc.org/G2-CR00220-change-generator-to-simpleContent

Note that a similar change was made made in NAR 1.2, according to comments eg on timestamp: `<!-- NAR 1.2 erratum 1 correction: on timestamp, complexContent replaced by simpleContent -->`

That raises the question, should this be an erratum release or a full release? We're not changing any functionality...

davecompton commented:
Hmm ... as it only contains non-functional / whitespace changes, it sounds like v2.34_2.
I guess that's a lot to rework at this point ... ?
